src/scenarios/shared/mauisharedpython.py

```diff
@@ -37,11 +37,11 @@ def generate_maui_rollback_dict():
         root = ET.fromstring(version_details_xml)
 
     # Get the General Band version from the Version.Details.xml file sdk version
-    general_version_obj = root.find(".//Dependency[@Name='VS.Tools.Net.Core.SDK.Resolver']")
+    general_version_obj = root.find(".//Dependency[@Name='Microsoft.NET.Sdk']")
```

Comment:
Why did this change?

Reply:
I am not able to find the high level PR/issue, but it was due to VS.Tools.Net.Core.SDK.Resolver no longer being supported with Microsoft.NET.Sdk being the replacement: https://github.com/dotnet/maui/pull/23967#issuecomment-2263510484.

Reply:
SGTM